calibration/NUTNRN/CGINS-NUTNRN-00367__20250312.csv

```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+367,CC_cal_temp,20.00,source_file 3305-00327-00256-A_SN_0367_Pre-Deployment_NUTNR-N.zip > SNA0367K.CAL; reference update conducted in Clark South Cold Room at 12C
```

Comment:
the pre-deploys above have the prefix NTR- before SN in 1st column, e.g., NTR-1103

Reply:
the glider/auv ones do not have NTR as part of SN